src/NodeDev.Blazor/Components/OptionsDialog.razor

```diff
@@ -0,0 +1,38 @@
+﻿@inject IDialogService DialogService
+@inject AppOptionsContainer AppOptionsContainer
+
+<MudDialog>
+    <TitleContent>
+        Options
+    </TitleContent>
+    <DialogContent>
+        <MudTextField @bind-Value="AppOptions.ProjectsDirectory" Label="Projects Directory" Variant="Variant.Text" AutoGrow></MudTextField>
```

Comment:
Since we're binding to the same object instance as the `AppOptionsContainer.AppOptions` (not a copy) we're gonna be changing the global value directly. Pretty sure if you test and "Cancel" the popup, the value in `AppOptionsContainer.AppOptions.ProjectsDirectory` will still be changed. You'll need to clone the object so that you're working on a separate instance

Reply:
FYI, that's why I had the ProjectSettings as a "record class", 'cause the cloning feature is already integrated in those